spec/lib/openid_connect/token_endpoint_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.